draft-ietf-taps-interface.md

```diff
@@ -1669,12 +1685,55 @@ Type:
 Default:
 : Disabled
 
-This property specifies how long to wait before deciding that a Connection has
-failed when trying to reliably deliver data to the destination. Adjusting this Property
+This property specifies how long to wait before deciding that an active Connection has
+failed when trying to reliably deliver data to the Remote Endpoint. Adjusting this Property
 will only take effect when the underlying stack supports reliability. The special value
 `Disabled` means that this timeout is not scheduled to happen. This can be a valid
 choice with unreliable data transfer (e.g., when UDP is the underlying transport protocol).
+See also {{keep-alive-timeout}}.
+
+### Timeout for keep alive packets {#keep-alive-timeout}
+
+Name:
+: keepaliveTimeout
+
+Type:
+: Numeric, with special value `Disabled`
+
+Default:
+: Disabled
+
+A transport system can be requested a protcol that supports sending keep alive packets {{keep-alive}}.
+This property specifies the maximum time an idle connection (one for which no transport
+packets have been sent) should wait before 
+the Local Endpoint sends a keep-alive packet to the Remote Endpoint. Adjusting this Property
+will only take effect when the underlying stack supports sending keep-alive packet. The special value
+`Disabled` means that this timeout is not scheduled to happen, and does not request the
+transport system to send keep-alive packets. Guidance on setting this value is 
+provided in {{!BCP145}}.
+
+### Limit for Keepalives {#keep-alive-count}
 
+Name:
+: keepaliveCount
```

Comment:
Not sure we need this as this might usually be the same as the usual retry count in other cases of loss...?

Reply:
I don't think it is the same number. I'm also not sure all Apps need the same number.

Reply:
What I'm saying is that this is rather a protocol decision than an application one. Why do I as an application that is anyway idle need to control when the connection will be declared failed?